src/activities/CategoriesEntity.js

```diff
@@ -50,4 +60,41 @@ export class CategoriesEntity extends Entity {
 
 		return selectedCategory;
 	}
+
+	equals(category) {
+		const selectedCategory = this.getSelectedCategory();
+		if (!selectedCategory) {
+			return category.categoryId === selectedCategory;
```

Comment:
Is this comparing `undefined === undefined`?

Reply:
Yeah. If there's no selected category then `category.categoryId` will be `undefined` as will `selectedCategory`